### DISCUSSION TITLE
Upload to the KDE Store

### DISCUSSION THREAD

**Discussion by Jak2k** [Donnnno/Arcticons-Linux#18] [Ideas]
Please upload the pack to the [KDE Store](https://store.kde.org/browse/). This allows users to easily discover, download and install the pack.

**Comment by autinerd** [comment]
I will look into it, thanks for the suggestion!

**Reply by Donnnno** [reply]
@autinerd since you're doing like all the work with maintaining the pack, feel free to make the call if you think we're ready to publish!

**Comment by Donnnno** [comment]
Good idea, but maybe we need to have a version with all the basics first, before we reach a bigger audience.

**Reply by Donnnno** [reply]
Although a beta release should be fine, and might help with expanding it and seeing what we're missing.

**Comment by kitters-kat** [comment]
Perhaps we need to figure out what these "basics" are and make a to-do list.

**Reply by Jak2k** [reply]
## Some of the basics I found are:

- [x] VS Code
- [x] Blender
- [x] Inkscape
- [x] LibreOffice
- [x] System Monitor (KDE)
- [x] GIMP

## An easy one should be (because already on android):
- [x] Obsidian

## KDE-Settings
- [ ] Appearance
- [ ] Workspace Behavior
- [ ] Shortcuts
- [ ] Search
- [ ] Notifications
- [ ] Regional Settings
- [ ] Accessibility
- [ ] KDE Wallet
- [ ] Online Accounts
- [ ] User Feedback
- [ ] Input Devices
- [ ] Display and Monitor
- [ ] Audio
- [ ] Color Management
- [ ] KDE Connect
- [ ] Printers
- [ ] Removable Storage
- [ ] Thunderbolt
- [ ] About this System
- [ ] Software Update

## Actions

These are mostly black/white, but have the wrong thickness


## Categories

- [ ] Education language
- [ ] Education math
- [ ] Education
- [ ] Engineering
- [ ] Science

## Devices

These are mostly black/white, but have the wrong thickness

## Emblems

- [ ] Music
- [ ] share
- [ ] link
- [ ] videos
- [ ] Rating stars

## Common File Types

- [ ] JSON
- [ ] XML
- [ ] RSS
- [ ] Archive
- [ ] Markdown

## Places

- [ ] folder-android
- [ ] folder-add
- [ ] folder_bookmark
- [ ] folder-downloads (folder-download is already there)

## Status

These are mostly black/white, but have the wrong thickness

## Others

- [ ] nvidia-settings & xorg-x11-drv-nvidia
- [x] fedora logo

**Reply by Donnnno** [reply]
Thanks for the overview! Sounds good to me, I'll see if I have time to make these, and then we can check out how to publish it :)

**Reply by kitters-kat** [reply]
Suggestion to add to the list:
GIMP

**Reply by Donnnno** [reply]
I've added a bunch of icons, but not the mappings (I don't have those, atm) here: https://github.com/Arcticons-Team/Arcticons-Linux/pull/30
